src/Mod/Draft/draftmake/make_hatch.py

```diff
@@ -23,26 +23,24 @@
 """This module contains FreeCAD commands for the Draft workbench"""
 
 import FreeCAD
-from draftobjects.hatch import Draft_Hatch_Object
+from draftobjects.hatch import Hatch
 from draftviewproviders.view_hatch import Draft_Hatch_ViewProvider
 
-def makeHatch(baseobject, filename, pattern, scale, rotation):
+def make_hatch(baseobject, filename, pattern, scale, rotation):
 
-    """makeHatch(baseobject, filename, pattern, scale, rotation): Creates and returns a
+    """make_hatch(baseobject, filename, pattern, scale, rotation): Creates and returns a
     hatch object made by applying the given pattern of the given PAT file to the faces of
     the given base object. Given scale and rotation factors are applied to the hatch object.
     The result is a Part-based object created in the active document."""
 
     if not FreeCAD.ActiveDocument:
         return
     obj = FreeCAD.ActiveDocument.addObject("Part::FeaturePython", "Hatch")
-    Draft_Hatch_Object(obj)
+    Hatch(obj)
```

Comment:
I think it's better to use a "namespace" name like Draft_Hatch

Reply:
It would be inconsistent to use "Draft_Hatch" as the name of the class. Not other Draft object class uses this prefix.

Reply:
hmmm, yes you are right... ok for using "Hatch" so.

Reply:
The viewprovider has an inconsistent name as well: "Draft_Hatch_ViewProvider". Renaming it seems to be problematic: existing Hatch objects break.

Reply:
Which objects? The ones created with current code?